src/poprox_recommender/default.py

```diff
@@ -245,6 +309,44 @@ def compute_similarity_matrix(todays_article_vectors):
                 similarity_matrix[i, j] = similarity_matrix[j, i] = np.dot(value1, value2)
     return similarity_matrix
 
+def find_topic(past_articles: List[Article], article_id: UUID):
+    # each article might correspond to multiple topic
+    for article in past_articles:
+        if article.article_id == article_id:
+            return [mention.entity.name for mention in article.mentions]
+
+def normalized_topic_count(topic_count_dict):
```

Comment:
I generally avoid putting types in variable names, so I'd write this with a type annotation to make it clearer what's in the dictionary:

```suggestion
def normalized_topic_count(topic_counts: dict[str, int]):
```